### PR TITLE
Added data list to homeview  component and added slight styling #12

### LINKED ISSUE #12: List the available users in home page
The home page should ask to get the full list of all the users from the backend (say `get://users/list`) in order to print a dynamic menu

### PR DESCRIPTION
**Pull request by VernitaJ**
Added the database list to the homeview component for better UI fixes #12

### CODE REVIEW

File: src/views/HomeView.jsx [hunk 1]
```diff
@@ -1,19 +1,38 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
 
-const HomeView = ({ match }) => {
+const HomeView = ({ match, uname }) => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
```
Comment (marcopeg) [inline src/views/HomeView.jsx]: Can you move this into a custom hook in the /state folder?

Something like `use-users-list.js`, that works like the `use-backend` (that we should rename to "use-user-data") or similar...

Comment (VernitaJ) [reply src/views/HomeView.jsx]: I understand. I'll work on that now.

File: src/views/HomeView.jsx [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+            <div key={user.id}>
+              <Link to={`/${user.id}`}>
+                <li>
+                  <h2 className="homeview-user-names">{user.id} </h2>
+                </li>
+              </Link>
+            </div>
+          ))
+        ) : (
+          <div>
```
Comment (marcopeg) [inline src/views/HomeView.jsx]: Can you put a `div` inside an `ul`? Or is it better to work this structure a bit differently?

File: src/views/HomeView.jsx [hunk 3]
```diff
@@ -1,19 +1,38 @@
...
+
+  useEffect(() => {
+    fetch(`https://m07.herokuapp.com/`)
+      .then(response => response.json())
+      .then(users => setUsers(users))
+      .catch(err => console.error(err));
+  }, []);
+
+  console.log("hey", users);
+
```
Comment (ChauPham90) [inline src/views/HomeView.jsx]: I wonder why from line 8 to line 15, you have moved from HomeView to src/use-users-list, but it is still showing here?

**Pull request merged**